drivers/include/mtd_spi_eeprom.h

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (C) 2220 ML!PA Consulting GmbH
```

Comment:
![Good Heavens, Just Look At The Time](https://user-images.githubusercontent.com/1301112/79605917-0fdf7d80-80f1-11ea-90f4-9076dc7d182d.gif)

Reply:
Best reaction ever :-D

Reply:
sent back from future :D